common/src/main/java/com/viaversion/viaversion/protocols/v1_20_3to1_20_5/rewriter/ComponentRewriter1_20_5.java

```diff
@@ -946,7 +942,15 @@ protected void convertHolderSet(final CompoundTag tag, final String name, final
         if (set.hasTagKey()) {
             tag.putString(name, set.tagKey());
         } else {
-            tag.put(name, new IntArrayTag(set.ids()));
+            final ListTag<StringTag> identifiers = new ListTag<>(StringTag.class);
+            for (final int id : set.ids()) {
+                final String identifier = Protocol1_20_3To1_20_5.MAPPINGS.blockName(id);
```

Comment:
Add a comment that it's okay to use the 1.20.3 block list because the new types are at the end